torch/csrc/jit/script/concrete_module_type.cpp

```diff
@@ -36,6 +36,11 @@ ClassTypePtr ConcreteModuleType::createNewTypeFromThis() {
     cls->addAttribute(name, type, isParameter);
   }
 
+  for (const auto& moduleInfo : modules_) {
```

Comment:
What is this adding to `cls`? Is `modules_` all callables attached to this type or is it just sub-`ScriptModule`s?

Reply:
it's all the submodules